src/mcts/search.cc

```diff
@@ -72,15 +72,9 @@ Search::Search(const NodeTree& tree, Network* network,
     pending_searchers_.store(params_.GetMaxConcurrentSearchers(),
                              std::memory_order_release);
   }
-  float strength_ratio =
-      options.Get<float>(SearchParams::kStrengthRatioId);
-  if (strength_ratio != 1.0f) {
-    // Factor to multiply execution time to get the delay to add, so that the
-    // execution time is strength_ratio^2 of the total. This was found to
-    // approximate linear strength scaling from strength_ratio 0 to 1.
-    delay_factor_ = 1.0f / (strength_ratio * strength_ratio +
-                            std::numeric_limits<float>::epsilon()) -
-                    1.0f;
+  float nps_limit = options.Get<float>(SearchParams::kNpsLimitId);
```

Comment:
I think it's better to do in params.cc (and it's fine for GetNpsLimit() to return `optional<float>` if that's needed).

params.cc is where all that happens anyway (and its purpose is actually to cache the parameters so that it won't have a performance impact), and search.cc is already a bit large.

Reply:
The optional bit was a remnant from an earlier iteration.